### PR TITLE
Feature/note editor

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Features added

- [x] Made add note functionality.
- [x] Update Note functionality.
- [x] Delete Note functionality.
  
*To access these feature you must be logged in*
Link to Preview : https://live-note-git-feature-note-editor-kevin-solomon.vercel.app/login
```email : adarshbalika@gmail.com```
```password : adarshBalika123```

### CODE REVIEW

**Comment by Jay-Gadhiya** [comment]
The preview link is not navigating to login

**Comment by Jay-Gadhiya** [comment]
-context coded well
-CSS coded well

**Comment by Jay-Gadhiya** [comment]
UI looks good

**Review by Jay-Gadhiya — APPROVED** [review]

File: src/util/deleteNote.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export const deleteNote = async (_id, token, noteDispatch) => {
+  const options = {
+    headers: {
+      authorization: token,
+    },
+  };
+  const response = await axios.delete(`/api/notes/${_id}`, options);
```
Comment (mohammed-sas) [inline src/util/deleteNote.js]: check response status before updating the state

File: src/util/saveNoteHandler.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      authorization: token,
+    },
+  };
+  try {
+    const response = await axios.post(
+      '/api/notes',
+      { note: { value, tags: ['asdsd'] } },
+      options
+    );
+    console.log(response.data);
```
Comment (mohammed-sas) [inline src/util/saveNoteHandler.js:15]: remove the log

File: src/util/updateNote.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+export const updateNote = async (_id, value, token, noteDispatch) => {
+  try {
+    const options = { headers: { authorization: token } };
+    const response = await axios.post(
+      `/api/notes/${_id}`,
+      { note: { value } },
+      options
+    );
+    noteDispatch({ type: 'ADD_NEW_NOTE', payload: response.data.notes });
```
Comment (mohammed-sas) [inline src/util/updateNote.js]: check response status

**Review by mohammed-sas — APPROVED** [review]
functionalities are working as desired
ui is looking good
Utilites function has been coded well
Other than suggested pr good to merge

File: src/util/updateNote.js (hunk 3 above)
Comment (Kevin-Solomon) [reply src/util/updateNote.js]: Will do the necessary changes thank you for pointing them out mohammed.

**Pull request merged**